src/elements/dialogs/session-details.html

```diff
@@ -171,7 +175,7 @@ <h3>{$ sessionDetails.speakers $}</h3>
         return getComputedStyle(this, `--${this.generateClassName(value)}`);
       }
 
-      _getCLoseBtnIcon(isLaptopViewport) {
+      getCloseBtnIcon(isLaptopViewport) {
```

Comment:
Why have you removed underscore?

Reply:
My mistake. I was changing `..CLose..` to `..Close..`

Reply:
ok :) just rename it, and you can merge. Going to approve